daemons/attrd/attrd_commands.c

```diff
@@ -865,7 +865,7 @@ attrd_peer_update(crm_node_t *peer, xmlNode *xml, const char *host, bool filter)
         free_xml(sync);
 
     } else if (safe_str_neq(v->current, value)) {
-        crm_info("Setting %s[%s]: %s -> %s from %s",
+        crm_notice("Setting %s[%s]: %s -> %s from %s",
                  attr, host, v->current, value, peer->uname);
```

Comment:
I am a little concerned that some resource agents (such as a node health monitor) can change attributes frequently, but we can try this and see if we get user feedback.

Since we're raising the visibility of this message, I'd like to take the opportunity to make it more user-friendly. First, if v->current or value is NULL, currently it will log "(null)" -- I think we should use "(unset)" instead.

Second, we have a special capability to show part of a message in both the system log and pacemaker detail log, and the rest only in the detail log. It works like

    fn("... " CRM_XS " ...")

where the part after CRM_XS will only be logged to the detail log. For this message we can put the "from" part after CRM_XS, since it's not particularly meaningful to an end user investigating a problem.